src/main/scala/Benchmarks/TriesBenchmark.scala

```diff
@@ -0,0 +1,82 @@
+package Benchmarks
+
+import java.io.File
+
+import Core.Dictionary.Implementations.Trie.Implementations.{MapEdgesFactory, ModularTrie}
+import Core.Dictionary.Implementations.{SetDictionary, TrieWordsDictionary}
+import Core.Dictionary.WordsDictionary
+import org.scalameter._
+
+import scala.collection.concurrent.TrieMap
+
+class LibTrieDict(trie: TrieMap[String, Boolean]) extends WordsDictionary {
+  override def containsStringThatStartsWith(prefix: String): Boolean = trie.contains(prefix)
+
+  override def containsFull(fullString: String): Boolean = trie.contains(fullString)
+}
+
+object TriesBenchmark {
+  val standardConfig = config(
+    Key.exec.minWarmupRuns -> 10,
+    Key.exec.maxWarmupRuns -> 10,
+    Key.exec.benchRuns -> 10,
+    Key.exec.independentSamples -> 10
+  ) withWarmer new Warmer.Default
+
+  def time[R](block: => R, name: String = "Elapsed time"): R = {
+    val last = standardConfig.measure({
+      block
+    })
+
+    println(s"$name: $last")
+    block
+  }
+
+  def testTrie(trie: WordsDictionary, name: String, search: Iterable[String]): Unit = {
+    time(search.foreach(word => {
+      trie.containsStringThatStartsWith(word)
+    }), name + " search")
+  }
+
+  def main(args: Array[String]): Unit = {
+    val dictFile = "./Dictionary/new_dict_without_yo_and_tire.txt"
+    val file = io.Source.fromFile(new File(dictFile))
+    val words = file.getLines().toArray
```

Comment:
Can braces be removed?

Reply:
getLines() is not a value. And *get* is an action, so braces informing that this is a method are suitable, imho.